examples/CMakeLists.txt

```diff
@@ -48,4 +49,12 @@ if(NOT MSVC AND NOT APPLE)
   target_link_libraries(dl_main YOMM2::yomm2 dl)
   target_link_libraries(dl_shared YOMM2::yomm2)
   add_test(NAME dlopen COMMAND dl_main)
-  endif()
+endif()
+
+if (NOT (MSVC AND YOMM2_SHARED))
+  # Running this example with a Windows DLL is too much of a hassle, because we
+  # would need to add the path to the directory containing yomm2.dll to PATH.
+  # Anyway, if it works with static linking, it is very unlikely that it fails
+  # with the runtime in a DLL.
```

Comment:
In recent CMake versions there is this generator expression that can help with DLL in case the example on Windows is desirable for completeness https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#genex:TARGET_RUNTIME_DLLS

Reply:
Supporting DLLs on Windows has cost me a huge amount of time and effort. I started work on v2, which will break backward compatibility. As a consequence of one of the changes (a small one for that matter) I plan to drop the shared library build, and have header-only the only option.  Also, I doubt few people will ever use the generator, and, those who do, will most likely use a trimmed down policy. What I pre-instantiate in the DLL contains everything, even if the consumer is built for release.